collectd/README.md

```diff
@@ -83,3 +84,23 @@ When the script successfully completes, it starts up collectd, which begins repo
 ## LICENSE
 
 SignalFx version of collectd is released under the Apache 2.0 license. See LICENSE for more details.
+
+## HTTP Proxy
+
+Collectd can talk through an http proxy if needed. The changes would need to be made on files that are sourced by the init scripts.
+
+On CentOS:
+* Modify/Create either of the following files: '/etc/sysconfig/collectd' or '/etc/default/collectd'
```

Comment:
This is what I get back from running the command on centos7 > systemctl show collectd

EnvironmentFile=/etc/sysconfig/collectd (ignore_errors=yes)
EnvironmentFile=/etc/default/collectd (ignore_errors=yes)

I'll check on centOS 6 and confirm.